docs/documentation/make-first-prototype/branching.md

```diff
@@ -32,8 +32,8 @@ Currently, the `juggling-balls` page sends the user directly to question 2. Inst
 // Run this code when a form is submitted to 'juggling-balls-answer'
 router.post('/juggling-balls-answer', function (req, res) {
 
-  // Make a variable and give it the value from 'juggling-balls'
-  var jugglingBalls = req.session.data['juggling-balls']
+  // Make a variable and give it the value from 'how-many-balls'
+  var jugglingBalls = req.session.data['how-many-balls']
```

Comment:
I think for consistency we should either use `how-many-balls`/`howManyBalls` everywhere or `juggling-balls`/`jugglingBalls` if that makes sense? What do you think?

Reply:
So making a change like this?

```suggestion
  var howManyBalls = req.session.data['how-many-balls']
```

Reply:
yeh and then when that variable is referenced in later lines